src/file-uploader/file-uploader.stories.ts

```diff
@@ -0,0 +1,184 @@
+import { Component, Input } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+
+import { action } from "@storybook/addon-actions";
+import {
+	withKnobs,
+	boolean,
+	text,
+	array
+} from "@storybook/addon-knobs";
+
+import { FileUploaderModule, NotificationModule, ButtonModule } from "../";
+import { NotificationService } from "../notification/notification.service";
+
+@Component({
+	selector: "app-file-uploader",
+	template: `
+		<ibm-file-uploader
+			[title]="title"
+			[description]="description"
+			[buttonText]="buttonText"
+			[accept]="accept"
+			[multiple]="multiple"
+			[(files)]="files">
+		</ibm-file-uploader>
+
+		<br><div [id]="notificationId" style="width: 300px"></div>
```

Comment:
If you're already using inline styles, you might as well not use a `br`.

```suggestion
		<div [id]="notificationId" style="width: 300px; margin-top: 20px"></div>
```